### PR TITLE
Finalize Zimlii

### LINKED ISSUE #989: Finalize ZimLII Migrations
# Checklist

* [ ] Homepage Menu List
* [ ] Hero Banner
* [ ] About Page Text/Link/Information
* [ ] About in Footer Text
* [ ] Newsletter Link if any
* [ ] Google Analytics
* [ ] Partner Logos on the homepage/With support from
* [ ] Terms of Use
* [ ] Contact Information

# Homepage Menu List
---
### Top Menu Bar

* Menu  Items- Judgements, Legislation, Covid-19 Legislation, Gazettes, Journals
* Add Link to About Page- see below for text for the page
* Language Options- This needs further discussion.

### Right Sidebar

* Add About Information

The Zimbabwe Legal Information Institute (ZimLII) provides free, full, and anonymous access to Zimbabwe case law, legislation, and legal books and journals.

* Add social media block defaulting tab is Twitter- refer to the old site
### Hero Banner Area

* Copy the text  from https://zimlii.org/

Welcome to the Zimbabwe Legal Information Institute
ZimLII publishes the law of Zimbabwe for free access to all

* High-Resolution Logos
AfricanLII, Laws.Africa, Legal Resources Foundation, Sweden Sverige, SIDA

### Footer
* About ZimLII

The Zimbabwe Legal Information Institute (ZimLII) is an independent not-for-profit Trust hosting an online repository of legal information from Zimbabwe and beyond. The organization aims to provide for the knowledge needs of a growing group of people learning, aware, and interested in the justice and legal framework of our country.  ZimLII is a member of the global Free Access to Law Movement. The organization works closely with other regional and international Legal Information Institutes (LIIs) to promote justice and rule of law by maximizing access at no cost to public legal information

* Add With Support From- Get this from the old site (https://zimlii.org/)
* New About page ( Link it to the top menu)

### Partners

AfricanLII
Legal Resources Foundation

### Contact Information
We get this from the old site.

### PR DESCRIPTION
**Pull request by actlikewill**
closes #989 

image

image

image

### CODE REVIEW

File: zimlii/static/images/lrf_logo.png
Comment (niiroobiro) [inline zimlii/static/images/lrf_logo.png:1]: Can this be more visible? @actlikewill

Comment (actlikewill) [reply zimlii/static/images/lrf_logo.png:1]: @niiroobiro I managed to find a better one

File: zimlii/templates/peachjam/_footer.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  </div>
+  <div>
+    <a href="https://laws.africa/" target="_blank" rel="noreferrer">Laws.Africa</a>
+  </div>
+  <div>
+    <a href="https://lrfzim.com/" target="_blank" rel="noreferrer">{% trans 'Legal Resources Foundation' %}</a>
+  </div>
+  <h6 class="mt-4">{% trans 'With support from' %}</h6>
+  <div>
+    <a href="https://sida.se/" target="_blank" rel="noreferrer">{% trans 'SIDA' %}</a>
```
Comment (actlikewill) [inline zimlii/templates/peachjam/_footer.html:51]: @niiroobiro could you confirm if the above url for SIDA is correct? `https://sida.se`

Comment (niiroobiro) [reply zimlii/templates/peachjam/_footer.html:51]: @actlikewill Yeah..Confirmed.That's the correct URL.

File: zimlii/static/images/lrf_logo.png
Comment (niiroobiro) [reply zimlii/static/images/lrf_logo.png:1]: great

**Review by nickmwangemi — APPROVED** [review]

**Review by nickmwangemi — APPROVED** [review]

**Pull request merged**